asf_heat_pump_suitability/getters/get_datasets.py

```diff
@@ -132,6 +132,21 @@ def get_df_osopen_uprn_latlon(**kwargs) -> pl.DataFrame:
     return df
 
 
+def load_gdf_historic_england_conservation_areas() -> gpd.GeoDataFrame:
+    """
+    Load GeoDataFrame with conservation area polygons from Historic England (CRS: EPSG:4326).
+
+    Returns:
+        gpd.GeoDataFrame: polygons of conservation areas in England
+    """
+    gdf = gpd.read_file(
+        config["data_source"]["E_historic_england_conservation_areas"],
+        engine="pyogrio",
```

Comment:
"For Geopandas versions <1.0, GeoPandas defaulted to use Fiona. Starting from GeoPandas version 1.0, the global default has changed from Fiona to Pyogrio.

The main reason for this change is performance. Pyogrio is optimized for the use case relevant for GeoPandas: reading and writing in bulk. Because of this, in many cases speedups >5-20x can be observed." 

^ taken from the [geopandas docs ](https://geopandas.org/en/latest/docs/user_guide/fiona_to_pyogrio.html):)

Reply:
Ah ok, so if it's the default is there a reason we explicitly set out `engine="pyogrio"`? We are using Geopandas version more recent than 1.0 right?

Reply:
You're right, it's not necessary to explicitly add the engine here because it should be used by default. Sometimes I put default params in explicitly anyway so it's clearer how the function works, especially for debugging :)